ibr/binToJs.js

```diff
@@ -23,9 +23,15 @@ function onChooseFile() {
             controls.update();
             controls.enablePan = false;
             controls.enableDamping = true;
-            var lines = IBRSDK.renderLayer( ibrData );
-            for (const line of lines) {
-                scene.add(line);
+            var structures = IBRSDK.renderStructure( ibrData );
```

Comment:
nit: renderStructure doesn't seem to be doing any rendering, maybe unpackStructure?

Reply:
good point!